src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -404,4 +428,17 @@ public static void append(final InputStream stream) throws IOException {
         return props;
     }
 
+    /**
+     * For a new key, adds a new list with one item otherwise adds the value to
+     * the existing list for that key.
+     * @param key The key for the attribute
+     * @param value The value for the attribute
+     */
+    private void addToMultiMap(final String key, final String value) {
+        this.multimap.putIfAbsent(
+            key,
+            new ArrayList<String>(1)
```

Comment:
@westonized this one here is not good. let's use a linkedlist instead since we don't know the size.